contentElements/MultiLineInputElement.js

```diff
@@ -70,8 +80,11 @@ MultiLineInputElement.prototype.init = function (variableName) {
  * This function is used recursively to retrieve an array with all modifiers.
  * @param {Array} modifiersArr - this is an array that holds all modifiers.
  */
+
 MultiLineInputElement.prototype.getAllModifiers = function (modifiersArr) {
+    modifiersArr.push(this.modifier());
     this.questionText().getAllModifiers(modifiersArr);
+    this.recalcHeight();
```

Comment:
I guess you accidentally added this.recalcHeight() here...
Please remove it and add it at the correct space.